src/diffpy/labpdfproc/tools.py

```diff
@@ -236,10 +237,10 @@ def load_user_info(args):
     the updated argparse Namespace with username and email inserted
 
     """
-    config = {"username": args.username, "email": args.email}
-    config = get_user_info(config)
-    args.username = config["username"]
-    args.email = config["email"]
+    check_and_build_global_config()
+    config = get_user_info(owner_name=args.username, owner_email=args.email)
+    args.username = config["owner_name"]
```

Comment:
is there a reason you are not using `config.get()` here?  That is usually better practice.